spec/requests/project_transfers_api_spec.rb

```diff
@@ -154,5 +154,56 @@
         let(:resource_id) {'notfoundid'}
       end
     end
+
+    describe 'Reject project transfer' do
+      let(:url) { "/api/v1/project_transfers/#{resource_id}/reject" }
+      let(:resource) { FactoryGirl.create(:project_transfer,
+                                          :with_to_users,
+                                          status: status,
+                                          to_user: current_user) }
+      let(:status) { :pending }
+      let(:payload) {{}}
+
+      describe 'PUT' do
+        subject { put(url, payload.to_json, headers) }
+        let(:called_action) { 'PUT' }
+
```

Comment:
For the sake of test coverage, please include the following for each describe section:
it_behaves_like 'a feature toggled resource', env_key: 'SKIP_PROJECT_TRANSFERS'